### ISSUE TITLE
A11y_Visual Studio Code Client_Copilot_Copilot inline chat_Non-text Contrast: While navigating with Keyboard the contrast ratio of focus is (1.787:1) which is less than required.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#203773]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.4.11; #Non-text Contrast; #A11yWCAG2.1;

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise (22H2) 
OS Build 22621.3007

### Repro Steps:
1.Open Visual studio insider
2. In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install.
3. Write a sample code and press CTRL+I to open the Copilot inline chat.
4. Observe that the contrast ratio of focus is on "Make request" button is (1.787:1) which is less than required.

### Actual:
While navigating with Keyboard tab key to the "Make request" button the contrast ratio of focus is (1.787:1) which is less than required.

### Expected:

While navigating with Keyboard Tab key to "Make request" button the contrast ratio of focus should be greater than or equal to required (3:1).

### User Impact:
Keyboard users will face difficulty while navigating through keyboard if the focus is not clearly visible to the user and ratio is less than required.

### Attachment:
![A11y_Visual Studio Code Client_Copilot_Copilot inline chat](https://github.com/microsoft/vscode/assets/98530069/56b84a6a-f7c5-4d4e-873a-2a559a6658c4)

**Comment by hbons** [comment]
The focus border now has increased contrast and is now the same colour as everywhere else in VS Code. We don't have any indication from users that this accent colour is a problem anywhere in the dark theme.
